src/esm_runscripts/config_control.py

```diff
@@ -0,0 +1,112 @@
+import yaml
+import esm_parser
+
+
+def check_configuration_consistency(config):
+    """ TODO: fill docstring
+    """
+    runscript_abspath = config["general"]["command_line_config"]["runscript_abspath"]
+    # TODO: can this be None or KeyError?
+    models = config['general']['valid_model_names']
+
+    with open(runscript_abspath, "rt") as f:
+        runscript = yaml.load(f, Loader=yaml.FullLoader)
+
+    for model in models:
+        model_config_path = config[model]["debug_info"]["loaded_from_file"]
+        # some models have multiple files. Take the one with <model>.yaml
+        if isinstance(model_config_path, list):
+            model_config_path = model_config_path[0]
```

Comment:
Maybe not the best check against index of 0. Models normally implement further reading, so those will be behind the actual model yaml in this list, yet this might not always be the case